source/_docs/configuration.markdown

```diff
@@ -35,7 +35,8 @@ If you run into trouble while configuring Home Assistant, have a look at the [co
 
 ## {% linkable_title Reloading changes %}
 
-You will have to restart Home Assistant for most changes to `configuration.yaml` to take effect. You can load changes to [automations](/docs/automation/), [customize](/docs/configuration/customizing-devices/), [groups](/components/group/), and [scripts](/components/script/) without restarting if you're not using [packages](/docs/configuration/packages/).
+You will have to restart Home Assistant for most changes to `configuration.yaml` to take effect.
+You can load changes to [automations](/docs/automation/), [customize](/docs/configuration/customizing-devices/), [groups](/components/group/), and [scripts](/components/script/) without restarting.
```

Comment:
Worth mentioning that it works with packages as well?

Reply:
I think we should only specify elements that do not work (like packages in the past).
Otherwise, we _should_ add that this works with config option 1 and 2, split configurations using `!include` and packages 🤷‍♂️

Reply:
I agree with Ludeeus things like this should be implied, and only things that are not compatible are worth mentioning.